uaclient/files.py

```diff
@@ -54,6 +54,10 @@ def read(self) -> Optional[str]:
     def delete(self):
         system.remove_file(self.path)
 
+    def create(self):
+        f = open(self.path, "x")
+        f.close()
```

Comment:
I think we can `Pathlib` touch here instead. If I am not mistaken, `Pathlib` should be available in python 3.5
But I think I also missing where in the code we are using this

Reply:
I noticed that with the retry file, I could not create the file only without having to call the `write` method. I thought having this method might be of help later, like that instance.

Reply:
Fair enough, but I missing where it is actually being called in the code. Are we actually using it ?

Reply:
Not currently. I noticed I do not need to create the retry file, but the caller of the api endpoint will create it.